src/main/java/net/consensys/orion/config/Config.java

```diff
@@ -100,6 +119,14 @@ private Config(final Configuration configuration) {
    * @return Port to listen on for the Orion API
    */
   public int nodePort() {
+    String port = env.get(envKey("nodeport"));
+    if (port != null) {
+      try {
+        return Integer.parseInt(port);
+      } catch (NumberFormatException e) {
+        log.warn(e.getMessage(), e);
+      }
+    }
```

Comment:
This logic seems to be reused in a few methods. I'd like to see a `getInteger(key)` method similar to `getString(key)`.

Reply:
done!